packages/bot-config-utils/test/fixtures/formats-wrong.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is this file used?

Reply:
No, I can remove. We had a JSON version of all the other fixtures too.